lambdas/build-stac/utils/stac.py

```diff
@@ -279,7 +280,11 @@ def from_cmr_links(cmr_links, item) -> Tuple[List, Dict[str, pystac.Asset]]:
                     "documentation", link["href"], link.get("type"), link.get("title")
                 )
             )
-
+        if link["rel"].endswith("browse#"):
+            asset = generate_asset(["thumbnail"], link, item)
+            if asset:
+                # name the asset after the href
+                assets[link["href"]] = asset
```

Comment:
I think we want either os.path.basename(link["href"]) or Path(link["href"]).stem
titles aren't typically the path to the object

Reply:
Ah I think I had misunderstood your slack message.

However, two base names can be the same theoretically for two different files, while the full URLs will never be the same.

Do you think it's problematic enough to change, redeploy and republish ?

Reply:
If the files were stored in different locations they could have the same file name, but I think that would be atypical when belonging to the same product? I was just going to make that edit today and republish

Reply:
Sometimes it's the full directory that identifies a file rather than the base name only, that's the edge case I thought of when doing that.

But I'm fine with either and it's pretty quick to change, it's just that we wanted to finish by this week. If you're okay to do it and merge, feel free :)